tests/test_main.py

```diff
@@ -67,6 +69,75 @@ def test_constructor():
     run_conda("constructor", "--help", check=True)
 
 
+@pytest.mark.parametrize("search_paths", ("all_rcs", "--no-rc", "env_var"))
+def test_conda_standalone_config(search_paths, tmp_path, monkeypatch):
+    expected_configs = {}
+    if recipe_dir := os.environ.get("PYINSTALLER_CONDARC_DIR"):
+        recipe_condarc = Path(recipe_dir, ".condarc")
+        if recipe_condarc.exists():
+            yaml = YAML()
+            with open(recipe_condarc) as crc:
+                recipe_config = YAML().load(crc)
+                expected_configs["recipe"] = recipe_config
+
+    config_args = ["--show-sources", "--json"]
+    if search_paths == "env_var":
+        monkeypatch.setenv("CONDA_RESTRICT_RC_SEARCH_PATH", "1")
+    elif search_paths == "--no-rc":
+        config_args.append("--no-rc")
+    else:
+        config_path = str(tmp_path / ".condarc")
+        expected_configs[config_path] = {
+            "channels": [
+                "defaults",
+            ]
+        }
+        with open(config_path, "w") as crc:
+            yaml.dump(expected_configs[config_path], crc)
+        monkeypatch.setenv("CONDA_ROOT", str(tmp_path))
+    env = os.environ.copy()
+
+    proc = run_conda(
+        "config",
+        *config_args,
+        check=True,
+        capture_output=True,
+        text=True,
+        env=env,
+    )
+    condarcs = json.loads(proc.stdout)
+
+    tmp_root = None
+    if recipe_dir:
+        # Quick way to get the location conda-standalone is extracted into
```

Comment:
Huh, is this stable across runs? I thought the name had some random characters everytime.

Reply:
It seems to be because I am using the `{{ RECIPE_DIR }}` environment variable. However, I decided to generalize the variable names.